src/etc/featureck.py

```diff
@@ -78,8 +78,15 @@
         if not filename.endswith(".rs"):
             continue
 
+        if sys.version_info.major == 2:
+            _open = lambda f: open(f, 'r')
+        elif sys.version_info.major == 3:
+            _open = lambda f: open(f, 'r', encoding="utf-8")
```

Comment:
it exploded on some of the innards of `Rc`, although I didn't dig into it super far.

I did a little reading, and the gist was that evidently python2 would lossily coerce invalid ascii characters, whereas python3 treats them as a hard error. I can dig up what I was reading if you'd like.